R/new-dataset.R

```diff
@@ -65,7 +65,7 @@ newDatasetFromFile <- function (file, name=basename(file), ...) {
         halt("File not found")
     }
     ds <- createDataset(name=name, ...)
-    ds <- addSourceToDataset(ds, createSource(file))
+    ds <- addSourceToDataset(ds, createSource(file), savepoint=FALSE)
```

Comment:
If it's truly a new argument that we want to support, then I'd like to see an integration test here that exercises the other case (savepoint=TRUE).